src/ghazal_azadi_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,45 @@
+from random import randint as rand_num
+
+__author__ = 'ghazalazadi'
+__email__ = 'ghaz@nmbu.no'
+
+
+def guessed_num():
+    """Returns the number being guessed by the user
+    """
+    inp_num = 0
```

Comment:
`inp_num` isn't a common style of creating abbreviated name. `input_num` or `in` are mainly used if you want to use the word `input`